grafana/grafana.bzl

```diff
@@ -59,7 +59,8 @@ py_dashboard = rule(
     attrs = {
         "builder": attr.label(
              executable=True,
-             cfg = "host",
```

Comment:
This forces the compiler to be python2 for all rules defined in this file, even if python_version is set to PY3.